engine/src/osxtheme.mm

```diff
@@ -152,6 +152,8 @@ static void converttomcrect(const Rect &macR, MCRectangle &mcR)
 	case WTHEME_TYPE_TAB:
 		return True;
 		break;
+    default:
+        break;
```

Comment:
Hang on this function is weird, it just returns `True` for everything.  Why not just change its whole body to `return True;`?

Otherwise surely its `switch` should cover all possible values of `Widget_Type` and return `False` for the ones not listed...?